spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/TraceFilter.java

```diff
@@ -271,6 +273,13 @@ private void detachOrCloseSpans(HttpServletRequest request,
 		}
 	}
 
+	/**
+	 * check exception is org.apache.catalina.connector.ClientAbortException.
+	 */
+	private boolean isClientAbortExpcetion(Throwable exception) {
```

Comment:
I don't like this. What if some other exceptions will be thrown? Will we have a list of exceptions for which we close the spans? I think we have to think of a different way of solving this problem.

Reply:
also typo. The javadoc should say why we are special-casing.. or a comment where we are special casing

Reply:
@marcingrzejszczak I will create a class that holds a list of exceptions that let the user configure which exceptions should be closed in this case. 
What do you think?